commands/management/add_mysql.go

```diff
@@ -111,4 +113,5 @@ func init() {
 
 	AddMySQLC.Flag("username", "MySQL username.").StringVar(&AddMySQL.Username)
 	AddMySQLC.Flag("password", "MySQL password.").StringVar(&AddMySQL.Password)
+	AddMySQLC.Flag("use-slowlog", "Run slow log agent instead of perf schema agent.").BoolVar(&AddMySQL.SlowLog)
```

Comment:
We need two separate flags – users may want to disable QAN completely.
Let's make both default to `false`. We will make an auto-detection later.

Reply:
ok